src/test/scenarios/datafactory/output/src/datafactory/README.md

```diff
@@ -1,5 +1,95 @@
-Microsoft Azure CLI 'datafactory' Extension
-==========================================
+# Azure CLI datafactory Extension #
+This is a extension for datafactory
 
-This package is for the 'datafactory' extension.
-i.e. 'az datafactory'
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name datafactory
+```
+
+### Included Features ###
+#### datafactory: ####
+datafactory
+* Examples:*
+```
+az datafactory create --location "East US" --zones "earth" --zones "moon" --name "myFactory" --resource-group "myResourceGroup"
+
+az datafactory show --name "myFactory" --resource-group "myResourceGroup"
+
+az datafactory list --resource-group "myResourceGroup"
+
+az datafactory update --name "myFactory" --tags exampleTag="exampleValue" --resource-group "myResourceGroup"
+
+az datafactory configure-factory-repo --factory-resource-id "/subscriptions/12345678-1234-1234-1234-12345678abc/resourceGroups/myResourceGroup/providers/Microsoft.DataFactory/factories/myFactory" --factory-vsts-configuration account-name="ADF" collaboration-branch="master" last-commit-id="" project-name="project" repository-name="repo" root-folder="/" tenant-id="" --location-id "East US"
+
+az datafactory get-data-plane-access --name "myFactory" --access-resource-path "" --expire-time "2018-11-10T09:46:20.2659347Z" --permissions "r" --profile-name "DefaultProfile" --start-time "2018-11-10T02:46:20.2659347Z" --resource-group "myResourceGroup"
+
+az datafactory get-git-hub-access-token --name "myFactory" --git-hub-access-code "some" --git-hub-access-token-base-url "some" --git-hub-client-id "some" --resource-group "myResourceGroup"
+
+az datafactory delete --name "myFactory" --resource-group "myResourceGroup"
+
+```
+#### datafactory trigger: ####
+datafactory trigger
+* Examples:*
+```
+az datafactory trigger create --factory-name "myFactory" --resource-group "myResourceGroup" --properties "{\\"type\\":\\"ScheduleTrigger\\",\\"pipelines\\":[{\\"parameters\\":{\\"OutputBlobNameList\\":[\\"exampleoutput.csv\\"]},\\"pipelineReference\\":{\\"type\\":\\"PipelineReference\\",\\"referenceName\\":\\"examplePipeline\\"}}],\\"typeProperties\\":{\\"recurrence\\":{\\"endTime\\":\\"2018-06-16T00:55:13.8441801Z\\",\\"frequency\\":\\"Minute\\",\\"interval\\":4,\\"startTime\\":\\"2018-06-16T00:39:13.8441801Z\\",\\"timeZone\\":\\"UTC\\"}}}" --name "myTrigger"
```

Comment:
Is there a way to process such long examples?
e.g.
```
az datafactory trigger create --factory-name "myFactory" \ 
--resource-group "myResourceGroup" --properties "...

```

Reply:
Currently now, need add this function.